### PR TITLE
SEO-180844-WinForms-refering-page

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @gowthamanthavasiyappan,
Kindly review the referring links issue fixes changes.

Thanks.

Regards,
Edith.

### CODE REVIEW

File: WindowsForms/Status-Bar/StatusBarAdv-Overview.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ documentation: ug
 
 The various sections of StatusBarAdv and their descriptions are given below.
 
-![Overview_img63](Overview_images/Overview_img63.jpeg) 
+![WindowsForm Status Bar overview.](Overview_images/windowsform-status-bar-overview.jpeg)
```
Comment (gowthamanthavasiyappan) [inline WindowsForms/Status-Bar/StatusBarAdv-Overview.md:14]: Windowsforms - change like this [both places]
image

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Status-Bar/StatusBarAdv-Overview.md (hunk 1 above)
Comment (Edithabich) [reply WindowsForms/Status-Bar/StatusBarAdv-Overview.md]: resolved

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Pull request merged**